types/ember/index.d.ts

```diff
@@ -3339,6 +3342,7 @@ declare module 'ember' {
          *                       containing a mapping of query parameters
          * @returns              the Transition object associated with this attempted transition
          */
+        transitionTo(routeNameOrUrl: string): Ember.Transition;
```

Comment:
This method should accept an optional `options` as well.
Could you also add no-model overloads to the other router service methods?

Reply:
Will do